test/cli/statsd-invalid-link/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# checking exception is raised when host is invlaid
+main/sorbet --silence-dev-message -e 'class Foo; end' --statsd-host="f4K#l1N&.com" --statsd-prefix=foo.bar 2> >(grep --color=never "statsd initialization failed")
```

Comment:
waiting for the day that someone registers this domain to break our CI 🙂

Reply:
`#` isn't allowed in domain names :p